lib/journaled/relation_change_protection.rb

```diff
@@ -1,7 +1,15 @@
 module Journaled::RelationChangeProtection
-  def update_all(updates, force: false)
+  def update_all(updates, force: false) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/PerceivedComplexity
     unless force || !@klass.respond_to?(:journaled_attribute_names) || @klass.journaled_attribute_names.empty?
-      conflicting_journaled_attribute_names = @klass.journaled_attribute_names & updates.keys.map(&:to_sym)
+      conflicting_journaled_attribute_names = if updates.is_a?(Hash)
+                                                @klass.journaled_attribute_names & updates.keys.map(&:to_sym)
+                                              elsif updates.is_a?(String)
+                                                @klass.journaled_attribute_names.select do |a|
+                                                  updates.match?(/(^|\A|[ \.\"])#{a}[\z \"$]/)
```

Comment:
I think this gets to the heart of what you're looking for, basically "is there an instance of #{a} that is neither proceeded (negative lookbehind) nor followed (negative lookahead) by a single quote but *is* surrounded by word boundaries?

```suggestion
                                                  updates.match?(/\b(?<!')#{a}(?!')\b/)
```

This means that `foo`, `"foo"`, `foo-bar` `foo,bar` will match, but `foo_bar`, `foobar`, and `'foo'` will not, which I think is good?

Reply:
yea, that seems good. thanks.